internal/namespaces/baremetal/v1/custom_server_create.go

```diff
@@ -60,7 +60,7 @@ func serverCreateBuilder(c *core.Command) *core.Command {
 		tmpRequest := argsI.(*baremetalCreateServerRequestCustom)
 		request := &baremetal.CreateServerRequest{
 			Zone:           tmpRequest.Zone,
-			OrganizationID: tmpRequest.OrganizationID,
+			OrganizationID: &tmpRequest.OrganizationID,
```

Comment:
this should set `OrganizationID` only is `tmpRequest.OrganizationID` is not empty